Automattic/woocommerce-payments
# Update UPE settings block for BNLP

Fixes #6276 

Update the UPE settings card to reflect Affirm and Afterpay as payment methods for those merchant who have the feature available.

The new design is tailored to also account for non-eligible merchants by only removing the icons from the asset.

The new design on desktop
![image](https://github.com/Automattic/woocommerce-payments/assets/24650518/76c69e5c-b3b1-4fdd-9127-29fb495acdbb)

The new design on Mobile
![image](https://github.com/Automattic/woocommerce-payments/assets/24650518/92407a52-61d9-43b7-8648-b17179445cf6)


#### Testing instructions

Hardcode the return value of `WC_Payments_Features::is_bnpl_affirm_afterpay_enabled()` method to both `true` and `false` and observe:
- if feature is not enabled, we should see the asset without "Affirm" and "Afterpay" icons
- if feature is enabled, we should see the asset with "Affirm" and "Afterpay" icons.

-------------------

- [ x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ x] Tested on mobile (or does not apply)

**Post merge**
